functests/ptp/prometheus.go

```diff
@@ -0,0 +1,126 @@
+package ptp
+
+import (
+	"encoding/json"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	client "github.com/openshift-kni/cnf-features-deploy/functests/utils/client"
+	"github.com/openshift-kni/cnf-features-deploy/functests/utils/pods"
+	k8sv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const openshiftPtpNamespace = "openshift-ptp"
+const openshiftPtpMetricPrefix = "openshift_ptp_"
+const openshiftMonitoringNamespace = "openshift-monitoring"
+
+type queryOutput struct {
```

Comment:
Can you add some comments here saying that these are needed to deserialize the query output (or in alternative, steal them from some prometheus package / give them more meaningful names)?

Reply:
Done